Core/src/main/java/com/plotsquared/core/command/DebugPaste.java

```diff
@@ -76,9 +78,9 @@ public boolean onCommand(final PlotPlayer<?> player, String[] args) {
                 StringBuilder b = new StringBuilder();
                 b.append(
                         """
-                         # Welcome to this paste
-                         # It is meant to provide us at IntellectualSites with better information about your problem
-                         """
+                                # Welcome to this paste
+                                # It is meant to provide us at IntellectualSites with better information about your problem
+                                """
```

Comment:
Unneeded change?

Reply:
Looks like Intellij did that while reformatting. Can change it back